packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -259,17 +254,17 @@ export default class SalesforceCRMService implements CRM {
   async handleEventCreation(event: CalendarEvent, contacts: Contact[]) {
     const sfEvent = await this.salesforceCreateEvent(event, contacts);
     if (sfEvent.success) {
-      this.log.debug("event:creation:ok", { sfEvent });
-      return Promise.resolve({
```

Comment:
Returning a value from an asynchronous function is assumed to be a resolved promise.